css/fonts.css

```diff
@@ -0,0 +1,7 @@
+@font-face {
+	font-family: 'GothamPro';
+	src: url('fonts/GothaProReg.eot');
+	src: local('☺'), url('../fonts/GothaProReg.woff') format('woff'), url('fonts/GothaProReg.ttf') format('truetype'), url('fonts/GothaProReg.svg') format('svg');
+	font-weight: normal;
+	font-style: normal;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Final-Axion-cite/pullRequest?prid=5980774)